netlify.toml

```diff
@@ -10,6 +11,7 @@
   publish = "out"
 
 [build.environment]
+  NETLIFY_NEXT_PLUGIN_SKIP = "true"
```

Comment:
so I understand that with this, the plugin that you use will only work for build cache improvements?

Reply:
@sambhavgupta0705

Reply:
oh ok, to skip ssr features, cool